packages/client/scripts/get-packed-client.js

```diff
@@ -9,4 +9,6 @@ async function main() {
   console.log(`Saving packed client to ${target}`)
 }
 
-main()
+main().catch((e) => {
+  throw e
+})
```

Comment:
After merging `main` back to this branch, `eslint` was throwing an error due to `@typescript-eslint/no-floating-promises`:

> Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator

Alternatively, this can be rewritten as

```suggestion
void main()
```